website/docs/components/alert/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,25 @@
+## Anatomy
+
+### Page and inline alerts
+
+![Anatomy of the page alert](/assets/components/alert/alert-anatomy-inline.png)
+
+| Element          |                                                 |
```

Comment:
This is probably a broader question (will be present in many other pages), but should there be a label in the column header? Probably a requirement for screen readers, could be something like "usage" or similar

Reply:
Yea, I realized that we can't do tables without headings in markdown, so I had to put something. I couldn't come up with anything for the second column, but I like "usage".